tensorboard/webapp/dev_assets/BUILD

```diff
@@ -40,19 +40,42 @@ tf_web_library(
     path = "/tb-webapp",
 )
 
+# By dropping 'compile', this skips the expensive minification step.
```

Comment:
We keep `compile` now; update comment?

(Or, is the comment needed now? This is in the `dev_assets` package, and
sets `dev_mode_only = True`, so is it self-explanatory? I defer to you.)